checkov/terraform/graph_builder/foreach/foreach_entity_handler.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import annotations
```

Comment:
@ChanochShayner can you highlight the specific changes done in this file?
Because it's a new file it's hard to tell what was just refactored out of the `ForEachResourceHandler` and what was changed

Reply:
No changes here at all, everything is the same in data and resources.